### ISSUE TITLE
Concate two gif into one

### ISSUE THREAD

**Issue by manoj99** [koral--/android-gif-drawable#292]
i want to joint multiple gif image into one.would you please tell me how it is possible?

**Comment by frmz** [comment]
You can probably just play a drawable once and then replace it with the other gif at the end of the loop

**Comment by manoj99** [comment]
i want functionality like multiple gif image joint each other not overlap each other.and how to save that result into sdcard.

**Comment by frmz** [comment]
as i said you can play one after the other, i am not the author but i don't think its the purpose of this library to reencode gifs or create them, you can probably find other libraries for that

**Comment by koral--** [comment]
Yeah, currently this library supports GIF decoding only, but encoding is planned to be added in the future.  I'll keep this issue open as a feature-request.

**Comment by zolga** [comment]
any update on gif encoding?

**Comment by koral--** [comment]
Unfortunately, not yet. But pull requests are always welcome.

**Comment by Adham-Mostafa99** [comment]
@koral--  any update for gif encoding ?

**Comment by koral--** [comment]
Not yet.
